tests/kickstart_tests/network-vlan.ks

```diff
@@ -0,0 +1,51 @@
+url --url="http://dl.fedoraproject.org/pub/fedora/linux/development/$releasever/$basearch/os/"
+install
+network --device=link --bootproto=dhcp
+# Create testing vlan interface
+network --device=link --vlanid=150 --bootproto=dhcp
+
+bootloader --timeout=1
+zerombr
+clearpart --all
+part --fstype=ext4 --size=4400 /
+part --fstype=ext4 --size=500 /boot
+part --fstype=swap --size=500 swap
+
+keyboard us
+lang en
+timezone America/New_York
+rootpw qweqwe
+shutdown
+
+%packages
+%end
+
+%post
+CONTENT=$(cat /etc/sysconfig/network-scripts/$(ls /etc/sysconfig/network-scripts/ | grep ".150"))
```

Comment:
I think you don't need to read the whole file here you can just get its name and then copy it. Then use the file name for the ``sed`` below to get the device. Also the second part of this could use a ``*`` pattern instead of the grep.